tlair/src/test/scala/at/forsyte/apalache/tla/lir/IrGenerators.scala

```diff
@@ -348,7 +348,7 @@ trait IrGenerators extends TlaType1Gen {
     for {
       ex <- exGen
       tt <- genTypeTag
-    } yield TlaAssumeDecl(ex).withTag(tt)
+    } yield TlaAssumeDecl(None, ex).withTag(tt)
```

Comment:
Should we generate named version?

Reply:
Ah, let's not worry about it for now. We can allow add that in a follow up. The generators are do for an overhaul at some point any how.